scripts/helpers/createExtensionManifest.mjs

```diff
@@ -327,6 +327,8 @@ const createExtensionManifest = ({ version }) => {
                       type: "integer",
                     },
                   },
+                  required: ["channel", "playerName"],
```

Comment:
Schema was to relaxed. You could have saved an object without channel or playerName. You could have saved also other properties that were not defined.